packages/piro/src/Piro_PerformAnalysis.hpp

```diff
@@ -78,6 +78,26 @@ namespace Piro {
      );
   //@}
 
+  //! \brief Performs analysis of a solved model using ROL.
+  //! \details Requires that the ROL package is available.
+  //! \ingroup Piro_Thyra_analysis_driver_grp
+  int PerformSROLAnalysis(
```

Comment:
I would call these: PerformROLSteadyAnalysis and PerformROLTransientAnalysis. Also the descriptions of the two functions need to be updated.